integration/rocket-chat/main.js

```diff
@@ -537,17 +537,17 @@ function SmartiWidget(element,_options) {
 
                     //map to dbsearch results TODO should be configurable
                     var docs = $.map(data.response.docs, function(doc) {
-                        return {
+                        /*return {
```

Comment:
@tkurz was that intentional that dbsearch is commented (and redlink search is active)